recipe_scrapers/streetkitchen.py

```diff
@@ -45,3 +53,44 @@ def yields(self):
         return get_yields(
             self.soup.find("span", {"class": "quantity-number"}).get_text()
         )
+
+    def category(self):
+        return self.soup.find("div", {"class": "entry-category"}).find("a").get_text()
+
+    def description(self):
+        return normalize_string(
+            self.soup.find("div", {"class": "entry-lead"}).get_text()
+        )
+
+    def author(self):
+        return normalize_string(
+            self.soup.find("a", {"rel": "author"}).find("img")["alt"]
+        )
+
+    def ingredient_groups(self) -> list[IngredientGroup]:
+        return group_ingredients(
+            self.ingredients(),
+            self.soup,
+            ".ingredients-main div.ingredient-group h3",
+            ".ingredients-main div.ingredient-group dd",
+        )
+
+    def prep_time(self):
+        items = self.soup.find("div", {"class": "the-content-div"}).find_all("li")
+
+        for item in items:
+            text = normalize_string(item.get_text())
+            if "Elkészítési idő" in text:
+                return int(text.split(" ")[-2])
+
+    def cook_time(self):
+        items = self.soup.find("div", {"class": "the-content-div"}).find_all("li")
+
+        for item in items:
+            text = normalize_string(item.get_text())
+            if "Sütési idő" in text:
+                return int(text.split(" ")[-2])
```

Comment:
Similarly for these I would recommend `get_minutes` if possible:

```suggestion
    def prep_time(self):
        items = self.soup.find("div", {"class": "the-content-div"}).find_all("li")

        for item in items:
            text = normalize_string(item.get_text())
            if "Elkészítési idő" in text:
                return get_minutes(text)

    def cook_time(self):
        items = self.soup.find("div", {"class": "the-content-div"}).find_all("li")

        for item in items:
            text = normalize_string(item.get_text())
            if "Sütési idő" in text:
                return get_minutes(text)
```